src/FullscreenQuad.cpp

```diff
@@ -0,0 +1,50 @@
+#include "FullscreenQuad.h"
+#include <Vertex.h>
+#include <RendererDX11.h>
+#include <ResourceManager.h>
```

Comment:
`" "` instead of `< >` for our headers

Reply:
Yeah, sorry about that. It's just that adding headers by pressing alt+enter (so I suppose it's intellisense that does that) always uses < > and sometimes I forget to change it.